<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeactivationDialog</class>
 <widget class="QDialog" name="DeactivationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>License Key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="licenseKeyEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Initialization Code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="initCodeEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="getCodeButton">
     <property name="text">
      <string>Get Deactivation Code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Deactivation Code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="activationCodeEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Hardware ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="hwidEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Confirmation code:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmationCodeEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="deactivateButton">
     <property name="text">
      <string>Deactivate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
